### PR TITLE
prepare for next major

### PR DESCRIPTION
**Pull request by kunicmarko20**
## Subject

Removing not needed stuff.

### CODE REVIEW

**Comment by kunicmarko20** [comment]
Can you block so someone doesn't merge, just realized I did not check typehint/returntypes

**Comment by greg0ire** [comment]
I think that this should be a separate PR

**Comment by kunicmarko20** [comment]
oh, ok then. Will do it after this is merged.

File: src/SonataCoreBundle.php [hunk 1]
```diff
@@ -146,10 +139,6 @@ public function registerFormMapping(): void
             'form.type_extension.form.data_collector',
         ];
 
-        if (class_exists(DescriptionFormTypeExtension::class)) {
-            $formTypes[] = 'nelmio_api_doc.form.extension.description_form_type_extension';
-        }
-
         FormHelper::registerFormExtensionMapping('form', $formTypes);
```
Comment (jordisala1991) [inline src/SonataCoreBundle.php]: I think everything related to FormHelper should get removed

Comment (kunicmarko20) [reply src/SonataCoreBundle.php]: There wasn't any kind of comment so I wasn't sure :confused:

Comment (jordisala1991) [reply src/SonataCoreBundle.php]: @greg0ire the formHelper is used only to keep compat with old form types declaration, or has more uses?

Comment (greg0ire) [reply src/SonataCoreBundle.php]: Not sure… maybe try removing it in a real app with form mapping disabled?

Comment (kunicmarko20) [reply src/SonataCoreBundle.php]: found the depreciation, will remove it now.

**Comment by kunicmarko20** [comment]
should be :ok_hand: now

**Comment by jordisala1991** [comment]
Do we need to modify the upgrade notes or something?

**Comment by kunicmarko20** [comment]
I don't think so, I think @Soullivaneuh does that when doing a releas, everything else should be already noted when deprecated?

**Comment by greg0ire** [comment]
Great job @kunicmarko20 !

**Pull request merged**